lib/nadgrid.js

```diff
@@ -10,11 +10,11 @@ var loadedNadgrids = {};
  * Load a binary NTv2 file (.gsb) to a key that can be used in a proj string like +nadgrids=<key>. Pass the NTv2 file
  * as an ArrayBuffer.
  */
-export default function nadgrid(key, data) {
+export default function nadgrid(key, data, includeErrorFields) {
```

Comment:
To make this safe to future upgrades, I'd change the signature to `nadgrid(key, data, options)`, with `options` currently only taking an `includesErrorFields` boolean option, with `true` being the default.